shared/src/packages/surveys/libs/validation-schemas/survey-request.validation-schema.ts

```diff
@@ -0,0 +1,15 @@
+import joi from 'joi';
+
+import { SurveyValidationMessage } from '../enums/enums.js';
+import { type SurveyRequestDto } from '../types/types.js';
+
+const SurveyValidationSchema = joi.object<SurveyRequestDto, true>({
```

Comment:
Let's rename it to CreateSurveyValidationSchema, but we will use it for both create and update
```suggestion
const CreateSurveyValidationSchema = joi.object<SurveyRequestDto, true>({
```